pkgs/by-name/as/assetto-server/package.nix

```
@@ -0,0 +1,30 @@
+{ fetchFromGitHub
+, buildDotnetModule
+, lib
+, dotnetCorePackages
+}:
+
+buildDotnetModule rec {
+  pname = "AssettoServer";
+  version = "0.0.54";
+
+  src = fetchFromGitHub {
+    owner = "compujuckel";
+    repo = pname;
```

```suggestion
    repo = "AssettoServer";
```

See https://github.com/NixOS/nixpkgs/issues/277994